addons/mail/models/mail_template.py

```diff
@@ -662,11 +665,13 @@ def send_mail(self, res_id, force_send=False, raise_exception=False, email_value
             force_send=force_send,
             raise_exception=raise_exception,
             email_values=email_values,
-            email_layout_xmlid=email_layout_xmlid
+            email_layout_xmlid=email_layout_xmlid,
+            subtitles=subtitles,
+            subtitles_highlight_2nd=subtitles_highlight_2nd,
         )[0].id  # TDE CLEANME: return mail + api.returns ?
 
     def send_mail_batch(self, res_ids, force_send=False, raise_exception=False, email_values=None,
-                  email_layout_xmlid=False):
+                  email_layout_xmlid=False, subtitles=False, subtitles_highlight_2nd=False):
```

Comment:
I suggest to have an ``additional_context`` (or ``layout_context``) that is propagated to ``_render_encapsulate``, now that it uses that method. That way we don't have to make the API bloated with details of rendering (everything is centralized in the render mixin, people can find details there).